### DISCUSSION TITLE
creating repository

### DISCUSSION THREAD

**Discussion by Bryium** [community/community#54941] [New to GitHub]
Despite my attempts to create a repository, I encountered an error message indicating that "something went wrong." Initially, I suspected the issue may be related to my browser, but after investigation, I am uncertain what may be causing the problem. Can you suggest any potential reasons for the error message and provide guidance on resolving the issue?

**Comment by iamreiyn** [comment]
Hi @Bryium 

There's an ongoing incident and major outage with several GitHub services, check [GitHub Status](https://www.githubstatus.com/) to read more, thank you.
